src/sagemaker/serve/model_server/torchserve/xgboost_inference.py

```diff
@@ -132,5 +138,30 @@ def install_package(package_name, version=None):
         print(f"Failed to install {package_name}. Error: {e}")
 
 
+def _get_mlflow_flavor():
+    mlflow_model_flavor = os.getenv("MLFLOW_MODEL_FLAVOR")
+    return mlflow_model_flavor
+
+
+def _load_mlflow_model(deployment_flavor, model_dir):
```

Comment:
Maybe add a todo to move the map to a utils file, so that it is a single source of truth...

Reply:
makes sense